src/components/Product.js

```diff
@@ -1,12 +1,25 @@
-import React from 'react'
-import styled from 'styled-components'
-import { formatPrice } from '../utils/helpers'
-import { FaSearch } from 'react-icons/fa'
-import { Link } from 'react-router-dom'
+import React from 'react';
+import styled from 'styled-components';
+import { formatPrice } from '../utils/helpers';
+import { FaSearch } from 'react-icons/fa';
+import { Link } from 'react-router-dom';
 
-const Product = () => {
-  return <h4>product</h4>
-}
+const Product = ({ image, name, price, id }) => {
+  return (
+    <Wrapper>
+      <div className="container">
+        <img src={image} alt={name} />
```

Comment:
```suggestion
        <img src={`https://ecommerce-6kwa.onrender.com/${image}`} alt={name} />
```

Reply:
Currently the value of `image` is something like: `"/upload/example.jpg"`. So currently the browser will just try to fetch that image from the same host that it's running on. So if you're running this React app locally on `localhost:3000`, it will try to do `localhost:3000/upload/example.jpg`. But that won't return anything since your React app doesn't actually know where the image is stored.

Instead, what you want is to get the image from your _server_ / backend app. Since your server is already deployed to Render.com, you can just get it from there (even when running locally).

Reply:
@akosasante you are so right! thank you

Reply:
@akosasante I am struggling here-how can I save my fotos on backend? at the moment I get them from unsplash or using direct link to google drive so `https://ecommerce-6kwa.onrender.com/${image}` doesn't work for me